Sources/Rendering/Misc/RemoteView/index.d.ts

```diff
@@ -126,7 +126,7 @@ export interface vtkRemoteView extends vtkObject {
    * 
    *  @param bypass
    */
-  setMouseScrollDebounceBypass(bypass: boolean): void;
+  setMouseScrollDebounceByPass(bypass: boolean): void;
```

Comment:
Can we just skip that method and call `remoteView.getInteractor().setMouseScrollDebounceByPass(true)` instead?